c/tests/test_tables.c

```diff
@@ -491,18 +491,18 @@ test_node_table(void)
     CU_ASSERT_EQUAL(table.metadata_length, 0);
 
     num_rows *= 2;
-    flags = malloc(num_rows * sizeof(uint32_t));
+    flags = malloc(num_rows * sizeof(tsk_flags_t));
     CU_ASSERT_FATAL(flags != NULL);
-    memset(flags, 1, num_rows * sizeof(uint32_t));
-    population = malloc(num_rows * sizeof(uint32_t));
+    memset(flags, 1, num_rows * sizeof(tsk_flags_t));
+    population = malloc(num_rows * sizeof(tsk_size_t));
```

Comment:
This and all the other id references below should be ``tsk_id_t`` I think.  Better again to use ``sizeof(*population)``, but that'd be more fiddly to do and not worth the effort I think.

Reply:
I've switched them to the right types - perils of search and replace!